content/en/docs/zero-code/obi/setup/kubernetes-helm.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Deploy OBI in Kubernetes with Helm
+linkTitle: Helm chart
+description: Learn how to deploy OBI as a Helm chart in Kubernetes.
+weight: 3
+---
+
+{{% alert title="Note" %}}
+
+For more details about the diverse Helm configuration options, check out the
+[OBI Helm chart options](https://github.com/grafana/beyla/blob/main/charts/beyla/README.md)
+document.
+
+{{% /alert %}}
+
+Contents:
+
+<!-- TOC -->
+
+- [Deploying OBI from helm](#deploying-obi-from-helm)
+- [Configuring OBI](#configuring-obi)
+- [Configuring OBI metadata](#configuring-obi-metadata)
+- [Providing secrets to the Helm configuration](#providing-secrets-to-the-helm-configuration)
+<!-- TOC -->
+
+## Deploying OBI from helm
+
+First, you need to add the Grafana helm repository to Helm:
+
+```sh
+helm repo add grafana https://grafana.github.io/helm-charts
+```
+
+The following command deploys a OBI DaemonSet with a default configuration in
+the `beyla` namespace:
+
+```sh
+helm install beyla -n beyla --create-namespace  grafana/beyla
+```
+
+The default OBI configuration:
+
+- exports the metrics as Prometheus metrics in the Pod HTTP port `9090`,
+  `/metrics` path.
+- tries to instrument all the applications in your cluster.
+- only provides application-level metrics and excludes
+  [network-level metrics](../../network/) by default
+- configures OBI to decorate the metrics with Kubernetes metadata labels, for
+  example `k8s.namespace.name` or `k8s.pod.name`
+
+## Configuring OBI
+
+You might want to override the default configuration of OBI. For example, to
+export the metrics and/or spans as OpenTelemetry instead of Prometheus, or to
+restrict the number of services to instrument.
+
+You can override the default [OBI configuration options](../../configure/) with
+your own values.
+
+For example, create a `helm-beyla.yml` file with a custom configuration:
+
+```yaml
+config:
+  data:
+    # Contents of the actual OBI configuration file
+    discovery:
+      instrument:
+        - k8s_namespace: demo
+        - k8s_namespace: blog
+    routes:
+      unmatched: heuristic
+```
+
+The `config.data` section contains a OBI configuration file, documented in the
+[OBI configuration options documentation](../../configure/options/).
+
+Then pass the overridden configuration to the `helm` command with the `-f` flag.
+For example:
+
+```sh
+helm install beyla grafana/beyla -f helm-beyla.yml
+```
+
+or, if the OBI chart was previously deployed:
+
+```sh
+helm upgrade beyla grafana/beyla -f helm-beyla.yml
+```
```

Comment:
~~~suggestion
```sh
helm install beyla grafana/beyla -f helm-beyla.yml
```

or, if the OBI chart was previously deployed:

```sh
helm upgrade beyla grafana/beyla -f helm-beyla.yml
```
~~~

we need to update these